bedrock/firefox/templates/firefox/os/mwc-2015-preview.html

```diff
@@ -5,7 +5,7 @@
 {% extends "firefox/base-resp.html" %}
 
 {% block page_title_prefix %}{% endblock %}
-{% block page_title %}{{_('Unleash the future - Firefox OS - MWC 2015')}}{% endblock %}
+{% block page_title %}{{_('Unleash the Future — Firefox OS — MWC 2015')}}{% endblock %}
```

Comment:
I snuck this change in: https://bugzilla.mozilla.org/show_bug.cgi?id=1120689#c17